book/version_control.md

```diff
@@ -155,53 +168,40 @@ Further paragraphs come after blank lines.
 
-Conflicts arise when multiple changes have been made to the same part of a file.
-You must indicate which change (or combination of changes) should be retained.
+### Versioning large files
 
-Don't panic!
-It's easy to make mistakes, but thankfully Git's audit trail means that we can always revert back to working versions.
-For most issues, [stackoverflow](https://stackoverflow.com/) is your friend.
+When versioning your repository, Git stores compressed copies of all previous versions of each file. Despite the file compression, this means that versioning very large or binary files quickly increase the size of your repository's history, especially if there are multiple versions of them. The size of your Git history determines how long it takes to `clone` or `pull` and `push` changes to and from your remote repository. This includes when a continuous integration platform downloads your repo to run tests and other checks. Therefore, storing large files in Git typically slows down your development workflow.
 
-Don't change published (i.e. `remote`) history, otherwise you might need to panic.
-This causes issues across different users of the repo, as other developer's local copies may no longer contain the same history.
-You must consider this before force pushing changes to a remote repository.
-Create new commits that resolve or `revert` to fix the problem.
+[Git Large Files Storage (LFS)](https://git-lfs.github.com/) is a Git extension that allows you to version large files, but without storing the files in your repository history. Large files in your repository's history are instead replaced with a small text-based pointer. This pointer references versions of the actual files, which are stored in a separate part of your remote repository (e.g. GitHub or GitLab). When you `pull` a repository including large files, only the current version of the file is retrieved from the remote server, rather than its whole history. This reduces the size of your local repository and the time taken to `push` and `pull` changes. [Git-LFS integrates well with a normal Git workflow](https://www.youtube.com/watch?v=uLR1RNqJ1Mw) and can be used for specific files, or even all files of a particular type within your repository.
+
+Other tools, including [git-annex](https://git-annex.branchable.com/) can be used for a similar purpose.
```

Comment:
`... or often third party storage (S3 buckets in the Cloud, SQL databases etc.) can provide an easy way to store data, although with varying levels of version control capability)'